service/discovery/azure/arm_template.go.tobemigrated

```diff
@@ -85,7 +81,6 @@ func (d *azureARMTemplateDiscovery) discoverARMTemplate() ([]voc.IsCloudResource
 	)
 
 	client := resources.NewGroupsClient(to.String(d.sub.SubscriptionID))
```

Comment:
We should avoid using the `to` package, since its part of autorest and we want to get rid of that